### PR TITLE
38739 fix api ga

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Description
Update api event names as per comments in ticket 38739

Tickets: 38739, 38858, [39945](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39945)

### CODE REVIEW

File: src/applications/personalization/dashboard/actions/debts.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const fetchDebts = () => async dispatch => {
       .filter(debt => debt.currentAr > 0)
       .map((debt, index) => ({ ...debt, id: index }));
     recordEvent({
-      event: `get-debts-successful`,
+      event: `api_call`,
       'error-key': `server error`,
```
Comment (adamwhitlock1) [inline src/applications/personalization/dashboard/actions/debts.js:49]: should this be set to `server error` if this is the event for success?

File: src/applications/personalization/rated-disabilities/actions/index.js [hunk 2]
```diff
@@ -22,21 +20,29 @@ export function fetchRatedDisabilities() {
       const errorCode = response.errors[0].code;
       if (isServerError(errorCode)) {
         recordEvent({
-          event: `${DISABILITY_PREFIX}-list-load-failed`,
-          'error-key': `${errorCode} internal error`,
+          event: `api_call`,
+          'error-key': `${errorCode} server error`,
```
Comment (adamwhitlock1) [inline src/applications/personalization/rated-disabilities/actions/index.js:24]: in the payments.js recordEvent it looks like the error key is `${errorCode}_server_error` should the error key for this and the client error also use underscores for consistency? I'm unfamiliar with the GA stuff in general, so just wondering if that would make errors easier to search in the GA dashboard or whatnot.

File: src/applications/personalization/appointments/actions/index.js [hunk 3]
```diff
@@ -164,7 +164,7 @@ export function fetchConfirmedFutureAppointments() {
             ],
           });
           recordEvent({
```
Comment (adamwhitlock1) [inline src/applications/personalization/appointments/actions/index.js:166]: I noticed the usage of `isClientError(error.status)` and 'isServerError` in some of the other handling. If that something that also needs to be checked here, or would this always result in a server error?

**Review by adamwhitlock1 — APPROVED** [review]
I had a few questions around the error codes in these and if they need to be the same or not, but otherwise LGTM

File: src/applications/personalization/dashboard/actions/debts.js (hunk 1 above)
Comment (tmitchellgcio) [reply src/applications/personalization/dashboard/actions/debts.js:49]: lol no. Let me fix that

**Pull request merged**